### DISCUSSION TITLE
Creating timesheet entries

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#3632] [Questions & Answers] [answered]
Hi all,

how could it be achieved that a person with role 'user' get's only his assigned activities when creating new timesheet entries?
Currently he gets all customers, projects and activities in the listboxes and he could create entries on activities/projects/customers he is  not assigned.

Best regards
Klaus

**Comment by kevinpapst** [comment, marked as answer]
With Teams: https://www.kimai.org/documentation/teams.html

**Reply by idemus** [reply]
Hi Kevin,

thanks for the hint. Teams are managing access on projects and customers content. But I still get all activities (not only my own ones assigned to my timesheet(s)). How is it managed that a person cannot choose from activities belonging to other team members (the different activities are needed because of  different rates inside the project).

As far as I understand a possible workaround could be having different projects (based on different rates inside the activities).

But maybe there is a way to manage access down to activities!?


Best regards
Klaus

**Reply by kevinpapst** [reply]
Yes, again: with Teams.

I don't know of any bug there. AFAIK all "something is not working with Teams bug messages" where related to an mis-configuration.
But I am open to be proven wrong.

Go ahead and create the not working configuration on the demo and let me know which users can be used to verify it.

**Reply by idemus** [reply]
Ok. No bug, you're correct and I've understood it, now. The creation of a timesheet and the assignment to an user has no influence to the visibility of the corresponding activity. So I have to create teams grouped by the different hourly rates inside the timesheet and assign them to the corresponding activity.

Thank you for your fast answers and for the wonderful application! :)
